definitions/json/AuthorisationCaseEvent.json

```diff
@@ -851,6 +851,12 @@
     "UserRole": "caseworker-employment-api",
     "CRUD": "CRUD"
   },
+  {
```

Comment:
This functionality needs to be available to caseworkers

Reply:
Refactored to enable caseworker functionality